miso-web/src/main/webapp/scripts/bulk_sequencingparameters.js

```diff
@@ -102,6 +102,13 @@ BulkTarget.sequencingparameters = (function () {
           data: "runType",
           maxLength: 255,
         },
+        {
+          title: "Movie Time",
```

Comment:
Update the Instrument Model field's `onChange` to enable/disable this as appropriate, similar to what it does for the ONT Run Type field. While you're at it, please add the missing space in the title "Instrument Model"